UnitTests/CommonTestUtils/MockExecutable.cs

```diff
@@ -1,47 +1,45 @@
 ﻿using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using GitExtUtils;
 using GitUI;
 using GitUIPluginInterfaces;
-using JetBrains.Annotations;
 using NUnit.Framework;
 
 namespace CommonTestUtils
 {
     public sealed class MockExecutable : IExecutable
     {
-        private readonly ConcurrentDictionary<string, ConcurrentStack<(string output, int? exitCode)>> _outputStackByArguments = new();
+        private readonly ConcurrentDictionary<string, ConcurrentStack<(string? output, string? error, int? exitCode)>> _outputStackByArguments = new();
```

Comment:
GetOutputLines() combines StandardOutput and StandardError already, so this adds another level to the tests that do not really exist.
GetOutputLines() should be changed to a standard method instead (possibly a "Raw" variant of Execute) and a helper function for combining the output lines.